ArduCopter/mode.h

```diff
@@ -282,7 +291,7 @@ class ModeAcro : public Mode {
     float throttle_hover() const override;
 
 private:
-
+    bool disable_air_mode_reset;
```

Comment:
I probably would have made this variable a positive to avoid the small developer confusion of deal with double negatives but it's OK.